cadl-tests/cadl/enum.cadl

```diff
@@ -0,0 +1,46 @@
+import "@cadl-lang/rest";
+
+using Cadl.Http;
+
+@serviceTitle("EnumService")
+@serviceVersion("1.0.0")
+namespace Cadl.EnumService;
+
+enum Color {
+  Red,
+  Blue,
+  Green,
+}
+
+@knownValues(Color)
+model ColorModel is string {
+}
+
+enum Priority {
+  High: 100,
+  Low: 0,
+}
+
+model Operation {
+    name: "Read" | "Write";
+    best: true;
```

Comment:
formatting

Reply:
and other LiteralType as well (except float as there is no distinguish in TS on int/float)